modules/httpcontrol/index.js

```diff
@@ -0,0 +1,123 @@
+import { EventEmitter } from 'events';
+import http from 'http';
+import querystring from 'querystring';
+
+/**
+ * Receives http requests from another peer.
+ */
+class HttpControl extends EventEmitter {
```

Comment:
Do we really need so much logs? Even if we need it maybe we should use lower level of logging for some of the messages. WDYT?

Reply:
Maybe info or debug ...